test/whitespace/extensions.unit

```diff
@@ -0,0 +1,108 @@
+40 columns                              |
+>>> indentation
+extension A on B {
```

Comment:
Should we include tests for extensions with omitted names?

https://github.com/dart-lang/language/blob/d30e0dfbecae35eef6b35dc20faad50a91eaf27b/accepted/future-releases/static-extension-methods/feature-specification.md#omitting-names-for-private-extensions